src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/RouteValueProvider.cs

```diff
@@ -52,6 +69,8 @@ protected PrefixContainer PrefixContainer
             }
         }
 
+        protected CultureInfo Culture { get; } = CultureInfo.InvariantCulture;
```

Comment:
The general pattern when you have an optional constructor argument is to have the shorter constructor overload call the longer one and pass in the default value that way, rather than setting it directly on a property.

While this will work, it's going to catch someone out who's reading the code.